src/components/App/App.js

```diff
@@ -10,19 +11,22 @@ import {Route, Redirect} from "react-router-dom";
 class App extends Component {
   constructor() {
     super();
+    this.mounted = false;
```

Comment:
Read an article about this - used for "cleanup" to cancel fetch requests when a component is unmounted. Helps with memory leaks.